docs/DeveloperGuide.md

```diff
@@ -340,8 +343,27 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use Case ends
 
-*{More to be added}*
+**Use case: Delete a student**
+
+**MSS**
+
+1. User requests to list students
+2. SETA shows a list of students
+3. User requests to delete a specific student in the list
+4. SETA deletes the person
+
+    Use case ends.
+
 
+**Extensions**
+
+* 2a. The list is empty
+  
+    Use case ends.
+* 3a. The given index is invalid.
+  * 3a1. SETA shows an error message.
+    Use case resumes at step 2.
+  
```

Comment:
I am not sure, but do we need to include all the use cases? It seems like you only included one here.